clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -475,11 +500,18 @@ public Node leastLoadedNode(long now) {
     }
 
     public static AbstractResponse parseResponse(ByteBuffer responseBuffer, RequestHeader requestHeader) {
+        return parseResponseMaybeUpdateThrottleTimeMetrics(responseBuffer, requestHeader, null, 0);
```

Comment:
@rajinisivaram : Thanks for the explanation. This makes sense. We can leave the code as it is. LGTM.